staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -176,6 +178,14 @@ func (mounter *Mounter) doMount(mounterPath string, mountCmd string, source stri
 	command := exec.Command(mountCmd, mountArgs...)
 	output, err := command.CombinedOutput()
 	if err != nil {
+		if err.Error() == errNoChildProcesses {
+			if command.ProcessState.Success() {
```

Comment:
are you able to test it with this scenario?

Reply:
I observed this on a customer prod cluster and couldn't reproduce it on my end so far. Seems to depend on the volume usage pattern and pod churn on their nodes.

Reply:
btw - rather than comparing strings I wonder if could have unwrapped the error and looked for `syscall.ECHILD` error specifically?